docs/api-reference/full-text/complex.mdx

```diff
@@ -4,7 +4,8 @@ title: Advanced Search Queries
 
 ## Overview
 
-The `search` function that's generated for your index can accept query objects instead of a query string. Nearly all the query types implemented in Tantivy, our underlying search engine, are exposed to Postgres.
+The `search` function that's generated for your index can accept query objects instead of a query string. Nearly all the query types implemented in Tantivy, our
+underlying search engine, are exposed to Postgres. If needed, you can directly refer to the [Tantivy documentation](https://docs.rs/tantivy/latest/tantivy/) as well.
```

Comment:
Remove, it's unclear how they're supposed to make use of the Tantivy docs or how we've mapped things.

Reply:
Is there a different place you think we could highlight this?

Reply:
I think sending our users to Tantivy will cause much more confusion than benefit.